sklearn/inspection/_plot/tests/test_boundary_decision_display.py

```diff
@@ -31,6 +36,12 @@
 )
 
 
+def load_iris_scaled():
```

Comment:
Please rename this to `load_scaled_2d_iris` to make the subsequent code less surprising.